rfcs/0006-merkle-tree/0006-merkle-tree.md

```diff
@@ -0,0 +1,145 @@
+---
+Number: 0006
+Category: Standards Track
+Status: Draft
+Author: Ke Wang
+Organization: Nervos Foundation
+Created: 2018-12-01
+---
+
+# Complete Binary Merkle Tree
```

Comment:
The title should be more specific, describe the ckb part for which this RFC is used instead of a simple name of data structure.